tensorflow/core/ops/ops.pbtxt

```diff
@@ -13604,6 +13604,74 @@ op {
     minimum: 1
   }
 }
+op {
```

Comment:
I believe you will need to update the api_def files here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/api_def/update_api_def.sh

Reply:
Sorry, I wrote the api_def files manually but forgot to 'git add' them. I just re-generated the files with the tool and add them to the new commit.